robotics/sensor/bms/BatteryPackStatus.0.1.uavcan

```diff
@@ -0,0 +1,114 @@
+# pack on a separate subject.
+#
+# The reported quantities are focused on the amount of energy that can be reclaimed from the battery. In a
+# simplified view, this can be seen as the amount of energy that is "stored" in the battery; however, this
+# interpretation is not strictly correct because the amount of retrievable energy may be dependent on external
+# factors such as the temperature of the battery or the load current. Energy estimation is hard and requires
+# accurate modeling of the state of the battery, which may be impossible to do without precise tracking of each
+# charging cycle. Despite the complications, this is considered to be a superior approach compared to the commonly
+# used alternative where the state estimation is focused on the electric charge, because the latter cannot be used
+# directly to predict the endurance of the system.
+#
+# The methods of energy estimation are implementation-defined.
+#
+# pragma: no-bit-length-limit
+#
+
+# The network-synchronized timestamp of the moment where the values contained in this message have been sampled.
+uavcan.time.SynchronizedTimestamp.1.0 timestamp
+
+# --------------------------  ESTIMATES  --------------------------
+
+void1
+
+# The instant SoC of the pack, ranges from 0 to 100. Mandatory.
+# The methods of estimation are implementation-defined.
+uint7 state_of_charge  # [percent]
+
+@assert _offset_ % 32 == {0}
+
+# Total instant pack voltage. Low-pass filtering should be applied to avoid aliasing effects.
+uavcan.si.unit.voltage.Scalar.1.0 voltage
+
+# Total instant pack current. Low-pass filtering should be applied to avoid aliasing effects.
+# Positive current flows from the battery to the load (discharging).
+# Negative current flows into the battery (charging).
+uavcan.si.unit.electric_current.Scalar.1.0 current
+
+# A pessimistic estimate of the amount of energy that can be reclaimed from the battery in its current state.
+# This may be dependent on the state of charge (SoC), state of health (SoH), temperature, load profile, etc.
+# Negative values may be reported to indicate overdischarge.
+#
+# The instant power consumption can be computed directly from the voltage and current measurements, or as
+# the time derivative of estimated energy (dE/dt). The latter approach takes into accout other environmental
+# factors that may affect the amount of reclaimable energy, so it may be preferable. Having the instant power,
+# the time to depletion is estimated as ((full_energy - energy) / power). When charging, the remaining time to
+# full charge can be found similarly.
+uavcan.si.unit.energy.Scalar.1.0 energy
+
+# A pessimistic estimate of the amount of energy that can be reclaimed from a fully charged battery
+# under the current conditions (SoH, temperature, load profile, etc).
+uavcan.si.unit.energy.Scalar.1.0 full_energy
+
+# If there are multiple temperature sensors, reduction is implementation-defined. NaN if not defined.
```

Comment:
Achieving two temperature values from multiple sensors: max_temperature and min_temperature would be more useful. 
In the case of one sensor, these values will be equal.

Reply:
Added.